requirements/dev.txt

```diff
@@ -8,4 +8,7 @@ parameterized==0.7.0
 pytest-xdist==1.30.0
 pytest-forked==1.1.3
 pytest-timeout==1.3.3
-pytest-rerunfailures==7.0
\ No newline at end of file
+pytest-rerunfailures==7.0
+
+# formatter
+black==20.8b1
```

Comment:
I like that we have this effectively defined, instead of relying on user to have a version of black installed. This has caused conflicts in the past, with files being formatted with different versions of black.